src/merkle/tiered_smt/tests.rs

```diff
@@ -721,13 +721,13 @@ fn tsmt_get_proof_single_element_64() {
     let mut smt = TieredSmt::default();
 
     let raw_a = 0b_00000000_00000001_00000000_00000001_00000000_00000001_00000000_00000001_u64;
-    let key_a = [ONE, ONE, ONE, raw_a.into()].into();
+    let key_a = [ONE, ONE, ONE, raw_a.try_into().unwrap()].into();
```

Comment:
Here, and in other similar places in this file, using `Felt::new()` would probably be easier than `.try_into().unwrap()`.